src/app/components/DishDescrip.jsx

```diff
@@ -0,0 +1,13 @@
+export default function DishDescrip() {
+  return (
+    <div className="dish__info">
+      <h2 className="js-dish-price">$32</h2>
```

Comment:
This doesn't seem a good way of using a heading tag
```suggestion
      <p className="js-dish-price">$32</p>
```